src/plugins/data/public/ui/dataset_selector/dataset_explorer.tsx

```diff
@@ -22,6 +22,8 @@ import { FormattedMessage } from '@osd/i18n/react';
 import { BaseDataset, DATA_STRUCTURE_META_TYPES, DataStructure } from '../../../common';
 import { QueryStringContract } from '../../query';
 import { IDataPluginServices } from '../../types';
+import { DataSetTable } from './dataset_table';
+import { DataStructureFetchOptions } from '../../query/query_string/dataset_service';
```

Comment:
nit: i would export this so from query_string and then query so that line 23 could be `QueryStringContract, DataStructureFetchOptions` incase other plugins would like to use this feature without the linter complaining about this not being exported from the top level folder of the plugin